meinberlin/apps/moderatorfeedback/models.py

```diff
@@ -27,23 +27,23 @@ class ModeratorStatement(UserGeneratedContentModel):
     )
 
     def save(self, *args, **kwargs):
-        self.statement = transforms.clean_html_field(self.statement)
+        self.feedback_text = transforms.clean_html_field(self.feedback_text)
         super().save(*args, **kwargs)
 
 
 class Moderateable(models.Model):
-    moderator_feedback_choices = DEFAULT_CHOICES
+    moderator_status_choices = DEFAULT_CHOICES
 
-    moderator_feedback = fields.ModeratorFeedbackField(
+    moderator_status = fields.ModeratorFeedbackField(
         verbose_name=_('Status (public)'),
         help_text=_(
             'The status appears below the idea or proposal in red, yellow or '
             'green. The creator of the contribution receives a notification.'
         )
     )
 
-    moderator_statement = models.OneToOneField(
-        ModeratorStatement,
+    moderator_feedback_text = models.OneToOneField(
```

Comment:
To me `moderator_feedback` makes more sense here? And then the text is `moderator_feedback.feedback_text` .. why did you name it `moderator_feedback_text`?

Reply:
Ah, but that is because you wanted a new name, right? Hm, alright.. still confuses me that its not named like the model.

Reply:
Oh yes, I thought it would be confusing as the moderator_status was formerly called moderator_feedback. And if sth. goes wrong during renaming or was forgotten, it would be worse?